scale-tests/templates/gensort-application.tmpl

```diff
@@ -39,7 +40,7 @@ spec:
     memory: "1g"
     labels:
       version: 2.4.3
-    serviceAccount: spark-driver
+    serviceAccount: spark-service-account
```

Comment:
Should service account name come from `gensort.sh` using environment variable? As user can change it by exporting `SERVICE_ACCOUNT_NAME` environment variable.
```suggestion
    serviceAccount: SERVICE_ACCOUNT_NAME
```

Reply:
Although we don't need a configurable service account for scale tests I updated the code to support the change you've suggested. The original reasoning was about having a hard-coded  static service account for all scale-test applications and create this account in `install.sh` script. But now it's templated as per your suggestion.